index.js

```diff
@@ -65,7 +67,11 @@ exports.parseQuery = function parseQuery(query) {
 	};
 	if(!query) return {};
 	if(typeof query !== "string")
-		return query;
+		return (util.deprecate(function () {},
+		"loaderUtils.parseQuery() just received a value type of " + typeof query +
+		" which can be problematic, see https://github.com/webpack/loader-utils/issues/56" + os.EOL +
+		"parseQuery() will be replaced with getOptions() in the next major version of loader-utils."))(),
+		query;
```

Comment:
Does it really have to be formatted this way??

The `,` operator is difficult to understand and I think it's better to use `{ ...; ...; }` here.